data/korath outfits.txt

```diff
@@ -99,7 +99,7 @@ outfit "Fuel Collector"
 	"outfit space" -20
 	"fuel capacity" 200
 	"ramscoop" 0.5
-	description "The Korath Fuel Collector is a large system combining a weak ramscoop with enough fuel capacity for one jump drive jump or two hyperdrive jumps."
+	description "Because of their habit to raid systems both far and near, Korath ships need a lot of fuel for their jump drives. The Fuel Collector is a bulky system which has a ramscoop to collect new fuel and a large capacity to store it."
```

Comment:
`Because of their habit of raiding systems near and far, Korath ships must carry a considerable amount of hyperspace fuel for their jump drives. The Fuel Collector is a bulky system that has both a ramscoop to harvest additional fuel, and a large capacity to store it.`

Also, "Fuel Collector" is a boring name.